.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Release
+on: 
+  release:
+    types: [published]
+
+jobs:
+
+  test:
```

Comment:
Avoid copy&paste: remove test job, assuming it's already run on push?

Reply:
This wasn't a copy'n'paste error but I intended! It proves that the commit going to be released passes all tests, to prevent mistakes...

Reply:
I did not imply a copy&paste error. I was just complaining about copy&paste at all.
As a result, it becomes tedious to maintain the test job twice.

Either you find out how to provide jobs in one place and include them. Or you do without the test job during the release, because it must have run during the previous commit anyway.

Reply:
Ok, I understood your point. I guess, I have idea how to achieve this.